npm-shrinkwrap.json

```diff
@@ -3500,258 +3500,320 @@
       "integrity": "sha512-cM+1OnKRacBh98mw5RJ3PEmFxhl70lDFAL50+x/a+fxzSZryBF1asADf57zliAtGhCNVHYzrJAZX+8R77AGYGg==",
       "dependencies": {
         "abbrev": {
-          "version": "https://registry.npmjs.org/abbrev/-/abbrev-1.0.9.tgz",
+          "version": "1.0.9",
+          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.0.9.tgz",
```

Comment:
> Ouch!

Haha, not really – with Sublime it's just press & hold <kbd>Cmd</kbd>+<kbd>D</kbd> on a selection that matches them all, edit, save, done ;)

> If it is, I guess a small reproducible example would be useful.

Yeah, I'll fiddle with it tomorrow